### ISSUE TITLE
Audit Trail

### ISSUE THREAD

**Issue by Danny-Talor** [Danny-Talor/SCE24-BioMedSW-Blood_Establishment-WPF#5]
An audit trail (also called audit log) is a security-relevant chronological record, set of records, and/or destination and source of records that provide documentary evidence of the sequence of activities that have affected at any time a specific operation, procedure, or event.

Audit records typically result from activities such as [financial transactions](https://en.wikipedia.org/wiki/Financial_transaction), [scientific research](https://en.wikipedia.org/wiki/Scientific_research) and health care data transactions, or [communications](https://en.wikipedia.org/wiki/Communication) by individual people, systems, accounts, or other entities.

TL;DR: log every action performed by the user
